db/helpers/common.js

```diff
@@ -143,7 +143,6 @@ function prepareToPublish(inst, changedKeys, ignoreAttributes) {
   for (let i = 0; i < changedKeys.length; i++) {
     if (!ignoreSet.has(changedKeys[i])) {
       return {
-        old: inst._previousDataValues,
         new: inst.get(),
```

Comment:
all the tests for the realtime events are under the realtime folder under tests.